sass/components/_header.sass

```
@@ -1,3 +1,17 @@
-img
-  max-height: 40px
-  max-width: 100%
+header
+  height: 80px
+  position: sticky
+  top: 0
+  background-color: white
+  z-index: 1
+  color: black
+  font-size: 0.875rem
+  font-family: 'Poppins',sans-serif
+
+  .header-menu
+    display: flex
+    float: right
```

```suggestion
    justify-content: flex-end
```
chị có thể tạo thêm mixin flex vì sau này cũng dùng nhiều ạ